checker/tests/nullness/Issue579Error.java

```diff
@@ -0,0 +1,17 @@
+// Addtional Test case for Issue579
```

Comment:
The "additional" will be clearer in the future if you directly reference "all-systems/Issue579.java"